### DISCUSSION TITLE
Have you tried to use Plaid iOS SDK ?

### DISCUSSION THREAD

**Discussion by Benoye** [ruffCode/yaba-kmm#65] [General]
Hi !

Thank you for this project, it is very interesting !
I am currently developing an app with KMM and I need to integrate Plaid on both iOS and Android.

However, I have some trouble adding the cocoapod dependency for Plaid on iOS.
Have you tried it yourself ?

**Comment by ruffCode** [comment]
Unfortunately, I haven't gotten to the iOS part of this app yet so I wouldn't be able to help
